code/gradle.properties

```diff
@@ -34,4 +34,4 @@ mavenRepoName=AdobeMobileEdgeBridgeSdk
 mavenRepoDescription=Adobe Experience Platform Edge Bridge extension for the Adobe Experience Platform Mobile SDK
 
 # Mobile Core dependency version
```

Comment:
could we clean this file up similar to the other extensions - e.g. https://github.com/adobe/aepsdk-edgeidentity-android/blob/staging/code/gradle.properties